libs/shared/src/dto/integration/shared.ts

```diff
@@ -0,0 +1,16 @@
+import { ChannelTypeEnum } from '../../entities/message-template';
```

Comment:
A note regarding the file name `shared.ts`, the entire library is called `@notifire/shared`, maybe better just to rename this as `create-integration.dto.ts` ?

Reply:
I will check it up! 🙃